ubuntu22.04/nvidia-driver

```diff
@@ -321,7 +351,18 @@ _load_driver() {
         _start_vgpu_topology_daemon
     fi
 
-    if _assert_nvswitch_system; then
+    if _assert_nvlink5_system; then
+        _ensure_nvlink5_prerequisites || return 1
```

Comment:
As mentioned yesterday, instead of returning an error here should we consider waiting until the drivers are loaded?

Reply:
Done, please re-review